tlm-stencil/src/components/mw-checkbox-button/mw-checkbox-button.tsx

```diff
@@ -0,0 +1,37 @@
+import {Component, getAssetPath, h, Prop} from '@stencil/core';
+
+@Component({
+  tag: 'mw-checkbox-button',
+  styleUrl: 'mw-checkbox-button.css',
+  assetsDirs: ['../assets'],
+  shadow: false,
```

Comment:
Why not use shadow DOM here?
I think we needed to disable it when using emotion/css